rules/object-spread.js

```diff
@@ -0,0 +1,8 @@
+// eslint-plugin-prefer-object-spread  - https://github.com/bryanrsmith/eslint-plugin-prefer-object-spread
```

Comment:
There´s no need to create have this file.

Could you move the activation of the rule into `addons/object-spread` instead? Just create a `rules` property and put the rule inside it.